test/main/store/migrations/index.test.js

```diff
@@ -931,3 +931,55 @@ describe('migration 25', () => {
     })
   })
 })
+
+describe('migration 26', () => {
+  beforeEach(() => {
+    state = {
+      main: {
+        _version: 20,
+        networks: {
+          ethereum: { 
+            5: {
+              id: 5,
+              type: 'ethereum',
+              layer: 'testnet',
+              symbol: 'ETH',
+              name: 'Görli',
+              explorer: 'https://goerli.etherscan.io',
+              gas: {
```

Comment:
I tend to just cut out all the parts of these objects that aren't needed to make the tests more tidy and readable, but will leave it to your discretion here

Reply:
I have cut out most of the junk, theres a little bit remaning as it gives a little context!